portfolio/src/main/webapp/script.js

```diff
@@ -72,10 +72,16 @@ function contact() {
 }
 
 /** Creates a map and adds it to the page. */
-function createMap() {
+function loadMap() {
+  var treeCSLatLng = {lat:37.430175, lng:-122.173388};
```

Comment:
I'm not clear on this variable name. What does treeCS mean (i get it's the gates computer science building, but future readers might not)?

Reply:
oh haha yeah I was thinking that using Stanford's mascot instead of the university's name would make the variable name more concise. but clarity is the number one priority-updated!